custom_model_runner/datarobot_drum/drum/adapters/model_adapters/python_model_adapter.py

```diff
@@ -746,6 +747,9 @@ def predict_unstructured(self, model, data, **kwargs):
         PythonModelAdapter._validate_unstructured_predictions(predictions)
         return predictions
 
+    def chat(self, model, completion_create_params):
```

Comment:
nit, more verbose param name:
```suggestion
    def chat(self, model, chat_completion_create_params):
```

Reply:
The param name was chosen to align with [type names in openai library](https://github.com/openai/openai-python/blob/1ed0e35679438cc7409f2e3aeb36d637b3c579c0/src/openai/types/chat/completion_create_params.py#L282):

Reply:
Got it 👍